### ISSUE TITLE
"Console Not Found"

### ISSUE THREAD

**Issue by littlermac** [X360Tools/PicoFlasher#14]
Hey there,
     I've used my Pico Flasher on two systems (a 4gb Corona and a Trinity), and it has worked great. But now, I am trying to use it for a third system (another 4gb Corona), and when I go to read the NAND in JRunner with Extras, I just get a "Console Not Found" error. I've tried this on another system (Another 4gb corona, the only other system I have around right now), and am getting the same error. It seems like another user here is having this issue as well but there seems to be no solution. I've checked my soldering several times as well as reflashed the pico a few times. Does anyone have any ideas as to what could be going on? Thanks!

**Comment by littlermac** [comment]
Yes, this is how I have it wired. It has worked on another 4gb Corona in the past.

**Comment by littlermac** [comment]
Hit the wrong button, sorry.

**Comment by Th3UnknownCoder** [comment]
Yup same issue here, PicoFlasher 3.0, my Pico only arrived today, I'd tested the soldering in my SD adaptor previously and it worked, was gonna leave this installed as a permanent nand reader for when I get the resistors I needed, but I legit can't do anything other than read "Console not found" with the Pico. I tested the wires with the SD adaptor again and it read absolutely fine, so im assuming its something to do with the Pico uf2 itself.

**Comment by shooter556** [comment]
"Not found" could occur if you didn't operate in a proper sequence.
Steps including: connecting wires from board to pico, connecting usb to computer, plug in the ac adapter to the board.
Though I can't remember the proper sequence, but you can just try yourself to shuffle them to find a good one.
Don't worry it would break anything if you try wrong. Just unplug all and try again.

**Comment by Th3UnknownCoder** [comment]
> "Not found" could occur if you didn't operate in a proper sequence.
> Steps including: connecting wires from board to pico, connecting usb to computer, plug in the ac adapter to the board.
> Though I can't remember the proper sequence, but you can just try yourself to shuffle them to find a good one.
> Don't worry it would break anything if you try wrong. Just unplug all and try again.

I spent 7 hours on this console yesterday, trust me I’ve tried every order possible.

**Comment by littlermac** [comment]
I have used the suggested order as well as several others as well.

**Comment by Th3UnknownCoder** [comment]
Spent another few hours working on the Corona I have this afternoon, checked continuity on pads which is fine, checked solder joints on Pico, all fine, resoldered the nand points, checked for continuity afterwards which is normal, tried several connection methods to get console to read, each time no console found. The same wires work fine in an SD Adaptor, I just don't wanna have to constantly use the SD card if I can leave the Pico in, but this is too much trouble.

**Comment by fabio296** [comment]
bom dia ,sou novo aqui queria relatar um problema que vem enfrentado  no pico ao fazer aleitura de uma corona 4gb ,console nao encontrado , sendo que fiz a soldagens tudo ok ,verifiquei varias vezes mas nao consigo fazer a leitura do mesmo..como posso resolver isso ??? obrigado

**Comment by AcidzDev** [comment]
Having the same issue with my picoflasher

Latest version of J-runner from Octal, Latest version of Picoflasher

2 Coronas both with Console not found. Verified RC2010 is present. Tried multiple usb cables, another pc. reflashed pico. Did the steps listed on Weekend modders site. 

I know my pico will do a 4GB corona as i have used it for one a few months ago, did a trinity with it a week before this issue on the coronas. Both consoles to boot to the dashboard. 

Not sure to what may be the issue, the only thing i have noticed is when I click the ? icon to identify the console J-runner freezes and gets a .NET error for operation time out. If i quit and try to reopen J-runner it takes about 5 mines to reopen it.


I will try to get the error log from J-runner as soon as i can.

**Comment by Seroczynski** [comment]
I have the same problem. Tested all connections, re-flashed Pico, ran flasher tool as admin. Still getting "Console Not Found" on my Corona 4GB.

**Comment by pablocool** [comment]
Same here. I have successfully flashed Corona v2 4GB few months ago. Now I am trying another and still "Console Not Found". Console boots to dash successfully and R2C10 is populated.

**Comment by pablocool** [comment]
Now I noticed that problematic console has PHISON controller but no POST_OUT. So is it v2 or v4? I thought v4 has NAND in front of MOBO and v4 PHISION + NAND on the back.

**Comment by pacman362** [comment]
I have 3 picos here all lasted 1-2 uses then do the no consul found ? would be nice to suss this out.

**Comment by pacman362** [comment]
> Hey there, I've used my Pico Flasher on two systems (a 4gb Corona and a Trinity), and it has worked great. But now, I am trying to use it for a third system (another 4gb Corona), and when I go to read the NAND in JRunner with Extras, I just get a "Console Not Found" error. I've tried this on another system (Another 4gb corona, the only other system I have around right now), and am getting the same error. It seems like another user here is having this issue as well but there seems to be no solution. I've checked my soldering several times as well as reflashed the pico a few times. Does anyone have any ideas as to what could be going on? Thanks!

Ok Update, fitted both my suspected FAKE Pico's with GENUINE RP2040's And im pleased to report they both work perfectly ! So in conclusion I suspect that the chips where indeed FAKE ! So all of you out there with dead Pico's Order yourselfs some new RP2040's and get reworking to bring them back from the dead for your RGH work.

**Comment by sweetlilmre** [comment]
Same issue, adding pull-up and pull-down as per: https://www.youtube.com/watch?v=_p8EKHUF9w0 solved it.

Doing this on the Pico side is easier and safer (for the pads):
Pull up: 10K resistor between Pin 11 (GP8) and Pin36 (3v3)
Pull down: Bridge Pin 27 (GP21) to Pin 28 (GND)

**Comment by Xerxes3rd** [comment]
Just wanted to note that I had to use the same pull-up and ground solution as @sweetlilmre.  I'm not sure I like bridging pin 27 to GND; I feel like it should go through some sort of resistor.  I assume that pin isn't driven high for very long or else it would result in a short?  Anyway, doing that made it work on the first try.  I did have to reset everything once between reading and writing.

**Comment by sweetlilmre** [comment]
@Xerxes3rd FYI: I tried this fork on a 4gb nand the other day and was able to flash using the SMC pins, not sure if this solves the not found issue as that xbox is long gone: https://github.com/15432/PicoFlasher
